guides/doc-Provisioning_Guide/topics/Cloud-GCE.adoc

```diff
@@ -42,15 +40,38 @@ To add a connection, complete the following steps:
 . From the *Zone* list, select the GCE zone to use.
 . Click *Submit*.
 
+.For CLI Users
+
+. In GCE, generate a service account key in JSON format and upload this file to the `/usr/share/foreman/` directory on {ProjectServer}.
```

Comment:
Not required to repeat as this is the same as UI.

What do you say ?

Reply:
I perceive those two procedures - the UI and CLI - as two different procedures. So I expect users who only use CLI to jump right to the CLI procedure and not look at the UI steps, that is why I repeat the same required steps in both procedures.

Reply:
Make sense ! ACK!

The change i m looking for might needs to be done globaly.